### PR TITLE
interim cloudformation revisions

### PR DESCRIPTION
**Pull request by jdmooney**
Interim cloudformation revisions and preliminary common https doc

### CODE REVIEW

File: userguide/tutorials/using-https.adoc [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+== Obtaining a Certificate
+
+There are many ways to obtain a certificate that you can use to protect a particular resource. Here we will describe three that may be of use in particular Kill Bill scenarios, especially those running on AWS. These are:
+
+1. Using a free CA (such as *Let's Encrypt*)
+2. Using the AWS Certificate Manager
+3. Creating a self-signed certificate
+
+
+== Using Certbot
```
Comment (sbrossie) [inline userguide/tutorials/using-https.adoc:55]: Maybe you should make it obvious that `Let's Encrypt` and `certbot` are the same thing, i.e this is the first option from what you mentioned above.

File: userguide/tutorials/using-https.adoc [hunk 2]
```diff
@@ -0,0 +1,126 @@
+= Using HTTPS and SSL/TLS
```
Comment (sbrossie) [inline userguide/tutorials/using-https.adoc:1]: Will this page be included for the Single AMI Deployments (`Single-Tier` and `Multi-Tier`) ?

File: userguide/tutorials/using-https.adoc [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+== CNAMEs
+
+A CNAME, or *Canonical Name*, is an identifier that links your CA to one or more resources that you are protecting with your certificate.
+
+== Obtaining a Certificate
+
+There are many ways to obtain a certificate that you can use to protect a particular resource. Here we will describe three that may be of use in particular Kill Bill scenarios, especially those running on AWS. These are:
+
+1. Using a free CA (such as *Let's Encrypt*)
+2. Using the AWS Certificate Manager
```
Comment (sbrossie) [inline userguide/tutorials/using-https.adoc]: If I understand correctly, you have only discussed option 1. below -- are you planning to add `2.` and `3.` as well, and then refer the various doc to point the relevant sections?

**Comment by jdmooney** [comment]
Stephane,

Thanks for your comments. The using-https doc is very much a work in 
progress. The idea is to try to consolidate all the discussion of https 
and certificates in one place. I'm not sure if this will work or not.

Jim

**Pull request merged**